change/@internal-react-components-25d9c1f2-2a56-460a-9b8b-7a11ae28996f.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "prerelease",
+  "comment": "Added new prop to chat message to allow custom date/time format",
```

Comment:
Adding prop name here would be helpful when changelog is created :)

Reply:
updated